utilities/tests/integration_tests/api/test_todoist_api_integration.py

```diff
@@ -2,28 +2,35 @@
 from typing import Dict, List, Optional
 
 import pytest
-from todoist_api_python.api import TodoistAPI
+from tests.conftest import DEFAULT_PROJECT, DEFAULT_SECTION
 
-from utilities.api.todoist_api import TodoistKeyError
+from utilities.api.base_classes.todoist import (
+    TodoistKeyError,
+    get_due_datetime_str,
+)
 
-PROJECT = "Pytest-area"
 
+class TestTodoistHelper:
```

Comment:
this is a unit test, do we want just this class used there & import it here for the actual integration test?

Reply:
looks fine to me as is. if it were imported from the unit tests then it would still have to be extended e.g. by deriving & overloading it. or maybe I misunderstand the point. I'd say we merge this & change it in the next MR if needed (to avoid issues).